x-pack/solutions/observability/plugins/observability/server/features/cases_v2.ts

```diff
@@ -36,12 +50,29 @@ export const getCasesFeatureV2 = (
         read: [observabilityFeatureId],
         update: [observabilityFeatureId],
         push: [observabilityFeatureId],
+        createComment: [observabilityFeatureId],
+        reopenCase: [observabilityFeatureId],
```

Comment:
I believe these should be removed as well based on the same reasoning as the `shared` privileges.